eng/Directory.Build.Common.targets

```diff
@@ -365,10 +365,11 @@
   <Target Name="GetPackageInfo">
     <PropertyGroup>
       <PackageIsNewSdk>false</PackageIsNewSdk>
-      <PackageIsNewSdk Condition="'$(IsClientLibrary)' == 'true'">true</PackageIsNewSdk>
+      <PackageIsNewSdk Condition="'$(IsClientLibrary)' == 'true' or '$(IsFunctionsLibrary)' == 'true'">true</PackageIsNewSdk>
```

Comment:
@weshaggard :  Does this have any impact on the artifacts json used by the smoke tests?
https://github.com/Azure/azure-sdk-for-net/blob/main/eng/scripts/smoketest/Get-Package-Version.ps1